db/schema.rb

```diff
@@ -71,6 +77,8 @@
     t.date "date_of_birth"
     t.text "ethnicity"
     t.text "disability"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
     t.text "address_line_one"
```

Comment:
I've noticed for some odd reason this attribute seems to change order for some reason after a db migrate